Front-End/challenge2/style.css

```diff
@@ -48,11 +48,11 @@ button {
 	background-color: #dbdcde;
 }
 
-#zero {
+/* #zero {
```

Comment:
remove commented code

Reply:
i will remove the comment. I just wanted to let you know that I commented that because I wanted the buttons to look the same size as I added the clear button